src/services/spotifyAuthenticationUrl.ts

```diff
@@ -1,5 +1,26 @@
-const { REACT_APP_CLIENT_ID, REACT_APP_FRONT_HOST } = process.env
+const { REACT_APP_SPOTIFY_CLIENT_ID, REACT_APP_FRONT_HOST } = process.env
 
+const spotifyUrl = new URL('https://accounts.spotify.com/authorize')
+if (REACT_APP_SPOTIFY_CLIENT_ID && REACT_APP_FRONT_HOST) {
```

Comment:
możesz dodać domyślną wartość REACT_APP_SPOTIFY_CLIENT_ID i REACT_APP_FRONT_HOST to wtedy nie  musisz warunku dodawać na wszsytkie parametry